config/environments/test.rb

```diff
@@ -29,7 +29,7 @@
   config.cache_store = :null_store
 
   # Render exception templates for rescuable exceptions and raise for other exceptions.
-  config.action_dispatch.show_exceptions = :rescuable
+  config.action_dispatch.show_exceptions = :none
```

Comment:
Tem algum motivo para terem alterado essa configuração?

Reply:
No rails 7.1 essa configuração mudou e começou a vir como padrão a opção :rescuable, sendo que, antes o padrão era :false. Aí a gente começou a encontrar alguns comportamentos estranhos do Capybara durante os testes. Por exemplo: Quando rodamos um teste que navegava para uma página, mesmo sem termos criado um controller, a mensagem de erro do teste não indicava a não existência do controller, só dizia que o conteúdo não existia naquela página (mesmo sem existir uma view nem um controller). Aí testamos a mesma url no navegador e o erro do navegador indicava a mensagem correta (uninitialized constant XController). Têm uma discussão sobre isso aqui:
https://discuss.rubyonrails.org/t/behaviors-in-action-controllers-raising-error-has-been-changed-in-7-1/83933